src/decision/src/subroutines/bottom_line/AdjustToLine.cpp

```diff
@@ -0,0 +1,44 @@
+//
+// Created by drdc-s2632 on 17/07/18.
+//
+
+#include "AdjustToLine.h"
+
+std::vector<ros::Subscriber>
+AdjustToLine::getSubscriptions(ros::NodeHandle nh) {
+    std::string gateDetectTopic = "line_detect_output";
+
+    std::vector<ros::Subscriber> subs;
+    subs.push_back(
+    nh.subscribe(gateDetectTopic, 10, &AdjustToLine::lineDetectCallback, this));
+    return subs;
+}
+
+void AdjustToLine::lineDetectCallback(
+const line_detect::LineDetectMsg::ConstPtr& msg) {
+    geometry_msgs::TwistStamped command;
+
+    if (msg->angleToParallelFrontMarker >
```

Comment:
would be great to have some general description of the intention around `if-else` blocks like these, just something like (if I interpret correctly)
```
// check yaw angle
if (angle > tolerance)
  ...
if angle < -tolerance
  ...
```
Just helps the push the reader in the right direction while reading/understanding

Reply:
done